Library/Formula/awscli.rb

```diff
@@ -96,6 +101,9 @@ def install
   end
 
   def caveats; <<-EOS.undent
+    Python >2.7.2 is recommended to prevent urllib3 SSL warnings:
```

Comment:
I'm -1 on adding this message, but don't feel that strongly about it if other maintainers don't.